index.html

```diff
@@ -4,23 +4,52 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+    <link rel="stylesheet" href="/style.css">
     <title>Document</title>
 </head>
-<body>
-    <h1>Awesome books</h1>
-    <div class="books">
-
+<body class="container-fluid w-100 d-flex flex-column align-items-center">
+  <nav class="navbar">
+    <a class="nav-link" href="#" id="add">Add new</a>
+    <a class="nav-link" href="#" id="list">List</a>
+    <a class="nav-link" href="#" id="contact">Contact</a>
+  </nav>
```

Comment:
- [ ] Your website navigation is working absolutely fine. Let's try to improve the design of the navbar. we can add a logo and nav link on the left and right end of the navbar respectively. Check the example below.

Yours
![image](https://user-images.githubusercontent.com/53019916/129388247-7b158939-8b2b-4ec4-80ca-4b111164118b.png)

Sample
![image](https://user-images.githubusercontent.com/53019916/129388427-923dbe49-5366-482f-840a-91c635832b6b.png)

Reply:
This issue is well resolved~ ✔️